include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -7085,10 +7085,8 @@ namespace VectorTools
         // if v is not a boring Vector or BlockVector:
         Assert(   dynamic_cast<Vector<double> *>(& v)
                   || dynamic_cast<Vector<float> *>(& v)
-                  || dynamic_cast<Vector<long double> *>(& v)
                   || dynamic_cast<BlockVector<double> *>(& v)
-                  || dynamic_cast<BlockVector<float> *>(& v)
-                  || dynamic_cast<BlockVector<long double> *>(& v),
+                  || dynamic_cast<BlockVector<float> *>(& v),
```

Comment:
I suspect that the change doesn't hurt -- but the code probably wouldn't be wrong with the old code either. Opinions?